ee/modules/110-istio/images/proxyv2-v1x16x2/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Based on https://github.com/istio/istio/blob/1.16.2/pilot/docker/Dockerfile.proxyv2
```

Comment:
```suggestion
# Based on https://github.com/istio/istio/blob/1.16.2/docker/Dockerfile.base
#      and https://github.com/istio/istio/blob/1.16.2/pilot/docker/Dockerfile.proxyv2
```
And double-check the dockerfiles please, they could be changed after migrating to ubuntu:focal in upstream.

Reply:
Why include a reference to the base image when it can be found in the original docker image?

Reply:
cheked